tests/kubectl_test.go

```diff
@@ -145,6 +146,47 @@ var _ = Describe("[sig-compute]oc/kubectl integration", func() {
 			Entry("[test_id:3466]virtualmachineinstance", "get", "vmi", "wide", []string{"NAME", "AGE", "PHASE", "IP", "NODENAME", "READY", "LIVE-MIGRATABLE", "PAUSED"}, 1, "True"),
 		)
 
+		It("should verify vmi ip value match primary interface ip value", func() {
+			Expect(createBridgeNetworkAttachmentDefinition(virtCli)).ToNot(HaveOccurred())
+
+			vm1 := tests.NewRandomVirtualMachine(tests.NewRandomVMI(), false)
```

Comment:
@EdDev

Reply:
I do not understand what this test adds here except of checking that an IP exists in the right column.
There is only one interface created, not two.

If it makes sense to check with at least two interfaces, then I would also assure that the status reports two before checking that an address appears in the output.


Syntactically, I would prefer `tests.NewRandomVMI()` to be replaced with:
```
libvmi.New(
	libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
	libvmi.WithNetwork(v1.DefaultPodNetwork()),
	libvmi.WithResourceMemory("1Mi")
)
```